docs/usage.rst

```diff
@@ -81,6 +81,26 @@ filesystem must be given as absolute paths. If paths are given that begin with `
 attempt to download the specified file or files from Google Cloud Storage. For this functionality, ``gcsfs``
 must be installed and authorized to download from the specified bucket.
 
+A ``field_table`` file is required for running the model, though it is not
+necessarily a required key in the ``config`` dictionary. In this file, names
+and properties of the different tracers used in the model are specified. If
+not provided in the ``config`` dictionary, ``fv3config`` will attempt to infer
+the required ``field_table`` based on the namelist parameters specified. This
+is typically recommended, because different ``field_table`` files are required
+depending on the model configuration, and it can be nice to rely on
+``fv3config`` to decide on what the appropriate file should be. That said, it
+can be useful to be able to override this. If one specifies a file path as a
+``field_table``, this file will be used regardless of the namelist
+parameters. If one specifies a directory as a ``field_table`` it will be
+assumed that the reference ``field_table`` files will be present in this
+directory, and will be used in place of the default reference ``field_table``
+files provided with ``fv3config``.  The current reference ``field_table`` files
+provided in ``fv3config`` correspond with files used for running with Zhao-Carr
+microphysics (``field_table_ZhaoCarr``) or with GFDL microphysics
+(``field_table_GFDLMP``). In FV3GFS these correspond with running with
+``gfs_phys_nml`` namelist parameters of ``imp_physics=99``, ``ncld=1``, and
+``imp_physics=11``, ``ncld=5``, respectively.
```

Comment:
Very descriptive, but so much text! Can I suggest something shorter?

> By default, fv3config attempts to automatically select the ``field_table`` file to use for the model based on the selected microphysics scheme in the namelist. This supports Zhao-Carr or GFDL microphysics. If the user provides a ``field_table`` key indicating a filename in the configuration dictionary, that file will be used instead.

If you want to come up with a different shorter paragraph feel free. If you think some of the extra "detail" is important you can add it as a [note/warning](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html). Also, this bit should probably go after the `namelist` is introduced in the paragraph below.

Reply:
Thanks for the suggested text.  I agree that's a nicer, more concise way of putting things.  I did add a note regarding the TKE-EDMF scheme, because I do think that's important for folks to be aware of, particularly when switching to the new fortran repo, which supports the new scheme, and is the scheme SHiELD is currently being run with at GFDL.